bifrost/renderer/+config.ts

```diff
@@ -13,5 +13,7 @@ export default {
     documentProps: { env: "server-and-client" },
     isLoggedIn: { env: "server-and-client" },
     scripts: { env: "server-and-client" },
+    storyblokSlug: { env: "server-only" },
+    onBeforeRender: { env: "server-and-client" },
```

Comment:
i wouldn't move onbeforerender to client. we want to do data fetching on the server since we will be hitting the private enki endpoints

Reply:
Oops, I think that was from when I was playing around with using it to run `storyblokInit` on server and client and forgot to remove that. Thanks for catching this